e2e/utils/test-utils.ts

```diff
@@ -187,28 +187,39 @@ export async function performAction(
 
 /**
  * Find an exsting workspace. Create a new workspace if none exists.
- * @param {boolean} createNew Create a new workspace, without regard to any existing workspaces or not.
+ * @param {boolean} create Create a new workspace, regardless any workspace exists or not.
+ * @param {string} workspaceName The name of a Workspace to find. If not found and create parameter is true, create new and returns it.
  */
-export async function findWorkspace(page: Page, createNew: boolean = false): Promise<WorkspaceCard> {
+export async function findWorkspace(page: Page, opts: {create?: boolean, workspaceName?: string} = {}): Promise<WorkspaceCard> {
```

Comment:
Adding one new feature in `findWorkspace` function: allow finding a known workspace. This is change needed for `owner-copy-to-workspace.spec` test to enable reusing same workspace to reduce test time. Because starting a new notebook server on a new workspace takes a long time, up to 10 minutes.